morph.js

```diff
@@ -21,15 +28,37 @@ function morph (newNode, oldNode) {
     const oldName = oldKv.name
     const oldVal = oldKv.value
 
-    if (newVal !== undefined) {
+    if (newVal === undefined) {
       removeAttribute(oldNode, oldName, oldVal)
     } else if (oldVal !== undefined || newVal !== oldVal) {
       setAttribute(oldNode, newName, newVal)
     }
   })
+}
 
-  if (newNode.nodeValue) oldNode.nodeValue = newNode.nodeValue
-  if (newNode.data) oldNode.data = newNode.data
+function copyAttrsNS (newNode, oldNode) {
+  const newAttrs = newNode._attributes
+  const oldAttrs = oldNode._attributes
+  const props = xtend(newAttrs, oldAttrs)
+
+  Object.keys(props).forEach(function (namespace) {
+    const vo = props[namespace]
+
+    Object.keys(vo).forEach(function (attrName) {
+      const newKv = newAttrs[namespace][attrName] || empty
+      const newName = `${newKv.prefix}:${attrName}`
```

Comment:
Good point! 
Felt like it was a bit more legible with the string templates, but needing a transpiler is gross.
As for `const` I was just following the coding conventions you had in place. Personally I don't use them since they can be misleading for people used to constants in other languages. Will refactor.